tests/components/deconz/test_lock.py

```diff
@@ -50,8 +49,9 @@ async def test_locks(hass):
     data = deepcopy(DECONZ_WEB_REQUEST)
     data["lights"] = deepcopy(LOCKS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
-    assert "lock.door_lock" in gateway.deconz_ids
+
     assert len(hass.states.async_all()) == 1
+    assert hass.states.get("lock.door_lock").state == STATE_UNLOCKED
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)